kolibri/plugins/coach/assets/src/views/common/TruncatedItemList.vue

```diff
@@ -42,4 +44,11 @@
 </script>
 
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+
+  .items-label {
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
```

Comment:
This looks great!

We're applying the `items-label` class to the entire translated string. I would expect it to just elide the entire translated string like this

> "Group 2, aaaaaaaaaaaaaaaaa..."


But in the screenshots it manages to individually elide the group names like like this:

> ![image](https://user-images.githubusercontent.com/2367265/52760190-733cbf80-2fc3-11e9-8427-0b4d92d4dd90.png)

How does this work?

Reply:
oh is it eliding on a line-by-line basis?